content/en/docs/concepts/storage/volumes.md

```diff
@@ -552,7 +552,7 @@ spec:
 <!-- maintenance note: OK to remove all mention of glusterfs once the v1.25 release of
 Kubernetes has gone out of support -->
 
-Kubernetes {{< skew currentVersion >}} does not include a `glusterfs` volume type.
+Kubernetes {{< skew currentPatchVersion >}} does not include a `glusterfs` volume type.
```

Comment:
The original is fine; this statement applies to all patch versions in a minor release.